llvm/include/llvm/ADT/SparseSet.h

```diff
@@ -205,7 +209,7 @@ class SparseSet {
     assert(Idx < Universe && "Key out of range");
     assert(Sparse != nullptr && "Invalid sparse type");
     const unsigned Stride = std::numeric_limits<SparseT>::max() + 1u;
-    for (unsigned i = Sparse[Idx], e = size(); i < e; i += Stride) {
+    for (unsigned i = Sparse.get()[Idx], e = size(); i < e; i += Stride) {
```

Comment:
If you make the `std::unique_ptr<SparseT, Deleter>` into a `std::unique_ptr<SparseT[], Deleter>` then you can use [] directly without the `.get()` I think?